src/components/ChartScale/ChartScale.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './ChartScale.css';
+
+const ChartScale = (props) => {
+    let width = 1000; // will need to change size based on grid/layout
+    let height = 600;
+    let labels = ['1 Billion', '2 Billion', '3 Billion', '4 Billion', '5 Billion', '6 Billion']; 
+
+    return (
+        <React.Fragment>
+            <div className="chart-scale--desktop">
+                <div class="label-container">
+                    {labels.slice(0).reverse().map((text, index) => <div style={{height: height / labels.length}} className="chart-scale__label" id={index}>{text}</div>)}
+                </div>
+                <div className="chart-scale--desktop-container" style={{maxWidth: width, height: height}}>
+                </div>
```

Comment:
I'm not sure there need to be two different dom elements for mobile / desktop. You can have one and change the display form block to flex, change the flex-direction, etc. using a media query. 

Also, I think that display: flex will take care of making all the child elements the same height/width, so you don't have to do that height/length calculation.

Reply:
One last question, what is the container element a container for? Is the idea that the Chart component goes in there?

Reply:
OK, I see what you mean about not having two elements for mobile/desktop! My thinking was that the ChartBar components would go in that container.